string    command # "clear_error", "torque_on", "torque_off", "reboot",  "enable", "disable" 
uint16[]  id_list
## "write", "read"　は未実装,要らないかも
# string    write_read_parameter
# float64[] write_value_list